### PR TITLE
Implements Goon's Gibbing Sounds

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Implement's Goon's better gibbing sound and adds in robot specific gibbing sound. Credit to Goon for these sounds

This just sounds better and more immersive than the one we currently have.

Volume has been adjusted a bit to compensate for the fact that this sound is innately louder than the one we currently use.

:cl: Fox McCloud
soundadd: Implement's goon's gibbing sound
soundadd: Implement's goon's gibbing sound for robots/silicons
sounddel: Removes old gibbing sound
/:cl:

### CODE REVIEW

**Comment by Spacemanspark** [comment]
You might want to put the actual sounds in the OP for those that haven't heard them.

**Comment by Asmerath** [comment]
sure, why the fuck not?

**Comment by Fox-McCloud** [comment]
>You might want to put the actual sounds in the OP for those that haven't heard them.

People can easily download and listen to them in the PR itself--no coding wizardry required for that.

**Comment by Spacemanspark** [comment]
No, but it's still kind of nice, and takes all of two seconds.

**Comment by tigercat2000** [comment]
mmmmmmmmmmmmm gibbing sounds

also
old: https://raw.githubusercontent.com/ParadiseSS13/Paradise/769c5c4ef1555c1359b5889f0252761e19dc21ed/sound/effects/gib.ogg
new: https://raw.githubusercontent.com/Fox-McCloud/Paradise-1/gibbing-sounds/sound/goonstation/effects/gib.ogg

**Comment by NTSAM** [comment]
do IPCs do the robogib noise or do they still just spark or wha

**Comment by Spacemanspark** [comment]
@NTSAM It looks like it's just cyborgs/ AIs.

**Comment by Fox-McCloud** [comment]
>do IPCs do the robogib noise or do they still just spark or wha

Nope. Just actual silicons.

**Comment by NTSAM** [comment]
oh

File: code/modules/mob/living/carbon/human/death.dm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 		animation.icon = 'icons/mob/mob.dmi'
 		animation.master = src
 
-		playsound(src.loc, 'sound/effects/gib.ogg', 100, 1, 10)
+		playsound(src.loc, 'sound/goonstation/effects/gib.ogg', 50, 1)
 
 	for(var/obj/item/organ/internal/I in internal_organs)
 		if(istype(loc,/turf))
```
Comment (Crazylemon64) [inline code/modules/mob/living/carbon/human/death.dm]: Line 35 or so of this file, is where you can bifurcate meatygibs and synthgibs, if you so desire.

**Pull request merged**